### ISSUE TITLE
Unable to go install v0.7.1

### ISSUE THREAD

**Issue by lox** [containrrr/shoutrrr#327]
Following the installation instructions in the README.md

```
go install github.com/containrrr/shoutrrr@latest
go: github.com/containrrr/shoutrrr@latest (in github.com/containrrr/shoutrrr@v0.7.1):
	The go.mod file for the module providing named packages contains one or
	more exclude directives. It must not contain directives that would cause
	it to be interpreted differently than if it were the main module.
```

Running go 1.19.4 on macOS, outside of my gopath.

**Comment by piksel** [comment]
I guess we need to drop compatibility with older go versions... It's excluded because it breaks building on go v1.16. I don't understand how go intended this to work...

**Comment by piksel** [comment]
This should be fixed as of #345 at least.
